crates/holochain/src/core/ribosome.rs

```diff
@@ -621,6 +621,7 @@ impl From<&ZomeCallHostAccess> for HostFnAccess {
 /// Interface for a Ribosome. Currently used only for mocking, as our only
 /// real concrete type is [`RealRibosome`](crate::core::ribosome::real_ribosome::RealRibosome)
 #[automock]
+#[allow(async_fn_in_trait)]
```

Comment:
Why the allow rather than the usual `#[async_trait]`?

Reply:
Oh I see the conversation above about this! That's great.

We still need an allow for it though?

Reply:
Yes, because it's a public trait https://blog.rust-lang.org/2023/12/21/async-fn-rpit-in-traits.html#--impl-trait-in-public-traits